StortSpelProjekt/Engine/src/Renderer/Bloom.cpp

```diff
@@ -55,7 +55,7 @@ Bloom::~Bloom()
 	delete m_pRenderTarget;
 }
 
-const RenderTarget* const Bloom::GetRenderTarget() const
+const RenderTargetView* const Bloom::GetRenderTarget() const
```

Comment:
GetRenderTargetView? Should it be GetRenderTarget? Name on return type and function type is not matching.

Reply:
Forgot to change the name.. The function is now named as GetRenderTargetView